submissions/tavor118sn/task_pre_oop/index.js

```diff
@@ -0,0 +1,71 @@
+const man = {
+  species: 'human',
+  name: 'Jon',
+  age: 25,
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  say: 'Winter is coming!',
+  friends: ['Ghost', 'Ygritte']
+};
+
+const woman = {
+  species: 'human',
+  name: 'Ygritte',
+  age: 25,
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  say: 'You Know Nothing, Jon Snow!',
+  friends: ['Jon']
+};
+
+const direwolf = {
+  species: 'direwolf',
+  name: 'Ghost',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  say: 'woof-woof!'
+};
+
+const cat = {
+  species: 'cat',
+  name: 'Tiger',
+  age: 1,
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  say: 'meow!',
+};
+
+const catWoman = {
+  species: 'human',
+  name: 'CatWoman',
+  age: 30,
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  say: cat.say,
+};
+
+const inhabitants = [man, woman, direwolf, cat, catWoman];
+
+const makeMessage = (obj) => {
+  let friends = '';
+  if (obj.hasOwnProperty(friends) && obj.friends.length > 0) {
+    friends = obj.friends.join(', ');
+  }
+
+  return [
+    `${obj.species}`,
```

Comment:
Please explain `` `${obj.species}` ``

Reply:
Not sure I understand the question. I need value of this key. I get it using dot notation.

Reply:
Or do you mean backquotes are redundant in this case?

Reply:
Why did you use backticks?

Reply:
2 reasons:
1) to make each line look similar 
2) it is easier to add some formatting/tags in the future
But I can remove if they are redundant.

Reply:
Clear, thank you. That's what I wanted to hear.